staking/app/api_utils.ts

```diff
@@ -120,35 +121,60 @@ export async function getAllMetadataAccounts(
   );
 }
 
-export async function getAllCustodyAccounts(
-  tokenProgram: any,
-  stakeAccounts: PublicKey[]
-) {
-  const allCustodyAccountAddresses = stakeAccounts.map((account) =>
-    getCustodyAccountAddress(account)
-  );
-  return tokenProgram.account.account.fetchMultiple(allCustodyAccountAddresses);
-}
-
 // ======================================
 // Locked accounts
 // ======================================
 
-export function hasStandardLockup(
-  metadataAccountData: IdlAccounts<Staking>["stakeAccountMetadataV2"]
-) {
-  return (
-    metadataAccountData.lock.periodicVestingAfterListing &&
-    metadataAccountData.lock.periodicVestingAfterListing.numPeriods.eq(
-      new BN(4)
-    ) &&
-    metadataAccountData.lock.periodicVestingAfterListing.periodDuration.eq(
-      ONE_YEAR
-    )
+function getAllLockedStakeAccounts() {
```

Comment:
can you leave a big warning comment here explaining that (1) uses a hardcoded list and (2) why it uses a hardcoded list? I think what you're doing is fine, and I get why you're doing it, but this is going to surprise people in the future.